build.sbt

```diff
@@ -274,7 +274,7 @@ lazy val lawsJS = laws.js
 
 lazy val free = crossProject.crossType(CrossType.Pure)
   .dependsOn(macros, core, tests % "test-internal -> test")
-  .settings(moduleName := "cats-free")
+  .settings(moduleName := "cats-free", name := "Cats Free")
```

Comment:
I just did it because it was named after the Free monad which I usually see capitalized. I can make it lower case if folks prefer that.